core/server/worker/src/main/java/alluxio/worker/grpc/DoraWorkerClientServiceHandler.java

```diff
@@ -118,7 +120,10 @@ public void listStatus(ListStatusPRequest request,
           request.getOptions().hasRecursive() ? request.getOptions().getRecursive() : false);
       UfsStatus[] statuses = mWorker.listStatus(request.getPath(), options);
       if (statuses == null) {
-        responseObserver.onError(new FileNotFoundException("File or Directory Not Found"));
+        responseObserver.onError(
+            new AlluxioRuntimeException(Status.NOT_FOUND,
```

Comment:
There is no NotFoundRuntimeException extends the AlluxioRuntimeException?
if not, can we add it?

Reply:
Oh, yes, there is a NotFoundRuntimeException!
I will fix it.

Reply:
Fixed in https://github.com/Alluxio/alluxio/pull/17107/commits/15593e76f2682ec9490a07a56e087b2e43fb8b95